WalletWasabi/Blockchain/BlockFilters/IndexBuilderService.cs

```diff
@@ -407,6 +407,14 @@ private void PrepareBech32UtxoSetHistory()
 			}
 		}
 
+		public FilterModel GetLastFilter()
+		{
+			using (IndexLock.Lock())
+			{
+				return Index[^1];
```

Comment:
This should be faster than `Index.Last()`. @lontivero ?

Reply:
It has to be exactly the same.

Reply:
Ack